product_sticker/views/product_sticker_views.xml

```diff
@@ -120,96 +120,74 @@
-                                        <div
-                                            t-if="record.product_category_id.value"
-                                            class="d-block"
-                                        >
-                                            <strong>Category:</strong>
-                                            <field name="product_category_id" />
-                                        </div>
-                                        <div
-                                            t-if="record.product_attribute_id.value"
-                                            groups="product.group_product_variant"
-                                            class="d-block"
-                                        >
-                                            <strong>Attribute:</strong>
-                                            <field name="product_attribute_id" />
-                                            <span
-                                                t-if="record.product_attribute_value_id.value"
-                                            > / </span>
-                                            <field name="product_attribute_value_id" />
-                                        </div>
+                                        <strong>Category:</strong>
+                                        <field name="product_category_id" />
                                     </div>
-                                    <div class="o_kanban_record_bottom">
-                                        <div class="oe_kanban_bottom_left" />
-                                        <div class="oe_kanban_bottom_right">
-                                            <field
-                                                name="create_uid"
-                                                widget="many2one_avatar_user"
-                                            />
-                                        </div>
+                                    <div
+                                        t-if="record.product_attribute_id.value"
+                                        groups="product.group_product_variant"
+                                        class="d-block"
+                                    >
+                                        <strong>Attribute:</strong>
+                                        <field name="product_attribute_id" />
+                                        <span
+                                            t-if="record.product_attribute_value_id.value"
+                                        > / </span>
+                                        <field name="product_attribute_value_id" />
                                     </div>
                                 </div>
-                            </div>
+                                <footer
+                                    class="position-absolute bottom-0 end-0 mb-2 me-2"
```

Comment:
Do you need to position it absolutely?

Reply:
Indeed maybe we don't need this information in the kanban card at all. What do you think, @Shide ?